src/modules/ce/components/Opportunity.tsx

```diff
@@ -50,81 +35,92 @@ const Opportunity: React.FC<Props> = ({}) => {
     return <NotFound />;
   }
 
-  const { activeReferral, acceptedReferral, name } = opportunity;
-
   return (
     <>
-      <PageTitle title={`Opportunity ${name}`} />
-      {!!(activeReferral || acceptedReferral) && (
-        <ButtonLink
-          to={generateSafePath(ProjectDashboardRoutes.REFERRAL_DETAILS, {
-            projectId: projectId,
-            opportunityId: opportunityId,
-            referralId: activeReferral?.id || acceptedReferral?.id || '',
-          })}
-        >
-          View Referral
-        </ButtonLink>
-      )}
-      {!activeReferral && !acceptedReferral && (
-        <Stack gap={2}>
-          <Typography variant='h4' component='h2'>
-            Candidates
-          </Typography>
-          <Paper>
-            <GenericTableWithData<
-              GetCeOpportunityCandidatesQuery,
-              GetCeOpportunityCandidatesQueryVariables,
-              CeCandidateFieldsFragment
-            >
-              columns={columns}
-              queryVariables={{ opportunityId: opportunityId }}
-              queryDocument={GetCeOpportunityCandidatesDocument}
-              pagePath='ceOpportunity.candidates'
-              paginationItemName='candidates'
-            />
-          </Paper>
-        </Stack>
-      )}
+      <PageTitle title={opportunity.name} />
+      <Typography variant='body2' mb={2}>
+        {project.projectName} Opportunity
+      </Typography>
+      <CommonTabs
+        currentTab={currentTab}
+        onChangeTab={(tab) => setCurrentTab(tab)}
+        ariaLabel={'Eligible Clients and Details tabs'}
+        tabDefinitions={[
+          {
+            title: 'Overview',
+            key: 'overview',
+            contents: (
+              <Grid container columnSpacing={2} rowSpacing={4}>
+                <Grid item xs={12}>
+                  <OpportunityBanner
+                    topCandidate={topCandidate}
+                    opportunity={opportunity}
+                    viewAllEligibleClients={() => setCurrentTab(1)}
+                  />
+                </Grid>
+                {opportunity.eligibilityRequirements && (
+                  <Grid item xs={12} md={6}>
+                    <MatchRuleGrid
+                      title='Requirements'
+                      rules={opportunity.eligibilityRequirements}
+                    />
+                  </Grid>
+                )}
+                {opportunity.priorityScheme && (
+                  <Grid item xs={12} md={6}>
+                    <MatchRuleGrid
+                      title='Prioritization'
+                      rules={[opportunity.priorityScheme]}
+                    />
+                  </Grid>
+                )}
+              </Grid>
+            ),
+          },
+          {
+            title: 'Eligible Clients',
+            key: 'clients',
+            contents: (
+              <Paper>
+                {/*todo @martha - discuss showing current referral and status here */}
```

Comment:
One thing from the [mocks](https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP/OP-HMIS-%2F-Coordinated-Entry-%2F-FigJam?node-id=2481-81423&t=iuUruj0pOsv7VX0j-0) that I haven't addressed at all: for in-progress and completed referrals, displaying the "eligible clients" list with that referral and its special status floated to the top.

I skipped it for now, since it introduces some complexity beyond just relying on `GenericTableWithData`, and the mocks are not finalized yet. Let me know if you think that sounds good, or I should put a little more time into it.

Reply:
I'm not sure what you're talking about here, can you share a screenshot?

I see that you did implement the "in progress referral" card on the overview page, so that seems sufficient.

Though in some mocks that card displays on the Eligible Clients tab:

<img width="738" alt="Screenshot 2025-03-24 at 9 35 28 AM" src="https://github.com/user-attachments/assets/5f9d7fd6-fa86-4964-ac22-be8eeba2f738" />

Reply:
I think lets leave it as-is

Reply:
Sorry for the unclarity! Here are screenshots of what I was trying to say.

Mocks show "status" column in "eligible clients" table:
![Screenshot 2025-03-24 at 10 44 00 AM](https://github.com/user-attachments/assets/911352fc-40d7-4a86-aa6a-160540923458)

Current implementation does not:
![Screenshot 2025-03-24 at 10 45 38 AM](https://github.com/user-attachments/assets/03910efa-9ae1-473a-a21d-935c28c5e9b3)

I'm good with leaving it as-is for now. I'll remove the todo.

Reply:
I see, yeah thats fine. thanks!